style.css

```diff
@@ -0,0 +1,187 @@
+/* Media Queries*/
+@media only screen(max-width:500px) {
+    #sidebar{
+        display: none;
+    }
+    .content-wrapper{
+        width: 100%;
+        padding: 0 25px;
+    }
+}
+
+
+body{
```

Comment:
- I would suggest you include a space after Ln5, put an indentation before the curly brace in Ln6 and remove the empty space in Ln12 for the cleanliness and readability of the code.